lib/rubocop/cop/layout/multiline_method_call_indentation.rb

```diff
@@ -193,7 +193,7 @@ def semantic_alignment_node(node)
           node = node.receiver while node.receiver
           # ascend to first call which has a dot
           node = node.parent
-          node = node.parent until node.loc.dot
+          node = node.parent until node.loc&.respond_to?(:dot) && node.loc&.dot
```

Comment:
Safe navigations looks unnecessary.

```suggestion
          node = node.parent until node.loc.respond_to?(:dot) && node.loc.dot
```